dbt/include/bigquery/macros/adapters/apply_grants.sql

```diff
@@ -11,10 +11,86 @@
 {% endmacro %}
 
 
-{%- macro bigquery__get_grant_sql(relation, privilege, grantee) -%}
+{%- macro bigquery__get_grant_sql(relation, privilege, grantees) -%}
+    {{ bigquery__get_grant_sql__option_3(relation, privilege, grantees) }}
+{%- endmacro -%}
+
+
+{%- macro bigquery__get_revoke_sql(relation, privilege, grantees) -%}
+    {{ bigquery__get_revoke_sql__option_3(relation, privilege, grantees) }}
+{%- endmacro -%}
+
+
+{%- macro bigquery__get_grant_sql__option_X(relation, privilege, grantee) -%}
+    grant {{ privilege }} on {{ relation }} to {{ grantees | join(', ') }}
+{%- endmacro -%}
+
+
+{%- macro bigquery__get_revoke_sql__option_X(relation, privilege, grantee) -%}
+    revoke {{ privilege }} on {{ relation }} from {{ grantees | join(', ') }}
+{%- endmacro -%}
```

Comment:
Default implementation in dbt-core, provided for comparison purposes.

Not an option, because this syntax doesn't work for BigQuery.